tools/base/aio.py

```diff
@@ -83,3 +105,405 @@ async def run_system_command():
         """
         loop = loop or asyncio.get_running_loop()
         return await loop.run_in_executor(executor, partial(subprocess.run, *args, **kwargs))
+
+
+_marker = object()
```

Comment:
Documentation please; this is some global exit status? Not sure what to make of the need for this.

Reply:
its a `sentinel`

Reply:
Maybe call it `_sentinel` as per https://www.python.org/dev/peps/pep-0661/ :)

Reply:
done, thanks for link, v useful